src/QsFmt/Parser/QSharpParser.g4

```diff
@@ -40,18 +40,20 @@ declarationPrefix : attributes+=attribute* access?;
 
 // Type Declaration
 
-typeDeclaration : declarationPrefix 'newtype' Identifier '=' underlyingType ';';
+typeDeclaration
+    : prefix=declarationPrefix keyword='newtype' declared=Identifier equals='=' underlying=underlyingType semicolon=';'
+    ;
 
 underlyingType
-    : typeDeclarationTuple
-    | type
+    : typeDeclarationTuple # TupleUnderlyingType
+    | type # OtherUnderlyingType
```

Comment:
nit: I'd like to find a more descriptive name for this other than `OtherUnderlyingType`, though I can see it is a bit tricky with this rules scheme. Maybe `UnnamedUnderlyingType` or `UnlabeledUnderlyingType` since this is the only way to get a type into the definition without an accompanying label. Though that also kinda sounds like an anonymous type, which is not what this is. Feel free to come up with a different name too; I just would rather find something other than `OtherUnderlyingType`.

Reply:
Thank you for seeing my struggle... The purpose of `typeDeclarationTuple` is to define types that [are similar to record types in F#](https://docs.microsoft.com/en-us/azure/quantum/user-guide/language/programstructure/typedeclarations). In the record type, there are named and unnamed items. So, I agree that `UnnamedUnderlyingType` or `UnlabeledUnderlyingType` would be confusing here. I have found the term "scala type" that is used as the opposite of "aggregate types" (maybe where "record type" belongs?). However, "scala type" mainly describes a scala value, which is also different from the "single-item type" semantics here. I was considering `NonRecordType`, but there is no context related to `record` here. Finally, I replace "record" with "tuple", and that is the current version.

Please replace it with a better name if you get one.

Reply:
Let's go with UnnamedTypeItem.

Reply:
No problem. It is done.